mac-linux/new-project

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Also does this file not need a .sh extension on the end of it? i.e. new-project.sh

Reply:
if the file had a .sh on the end then the user would have to run `new-project.sh` as the command and not `new-project`. not having the extension on the end really doesnt do anything.

Reply:
Okay, awesome! Good to know! Thanks for all the help!